modules/lvms-troubleshooting-recovering-from-missing-lvms-or-operator-components.adoc

```diff
@@ -67,9 +48,14 @@ topolvm-node-dr26h                    4/4     Running   0             66m
 vg-manager-r6zdv                      1/1     Running   0             66m
 ----
 +
-The expected output is one running instance of `lvms-operator` and `vg-manager`. One instance of `topolvm-controller` and `topolvm-node` is expected for each node.
+The output of this command must contain a running instance of the following pods:
+
+* `lvms-operator`
+* `vg-manager`
+* `topolvm-controller`
+* `topolvm-node`
```

Comment:
```suggestion
```
This is no longer the case for 4.16 and beyond, but still necessary for lower versions

Reply:
As there are two updates specific to 4.16+, I will make note of these and create another PR to make these updates only in 4.16.